frontend/src/views/CensusDirectory/index.tsx

```diff
@@ -21,7 +27,10 @@ import {
   textSecondary,
 } from "src/common/theme";
 
-function DetailItem(props: { label: string; children: string }) {
+import staticProjects, { type StaticProject } from "census-projects.json";
+import notebookLinks from "census-notebook-links.json";
+
+function DetailItem(props: { label: string; children: string; link?: string }) {
```

Comment:
Possible to extract styles to a separate style file?

Reply:
this is done in a following PR